index.html

```diff
@@ -123,7 +123,7 @@ <h2>Multi-Post
 
         <section class="project-section">
             <figure>
-                <img src="./Images/gym_fit.png" alt="" id="snap-shot-image">
+                <img src="./Images/gym_fit.png" alt="" class="snap-shot-image">
             </figure>
             <section class = "project-showing">
                 <h2>Tonic</h2>
```

Comment:
- [ ] Kindly 😊 try to decrease the margin between the items in the section here to make it nice like the Figma Design as you see here the margin between the items inside the section is greater than the Figma Design ,so let's try to make it nice like the Figma Design 🚀👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/235674143-714dddff-3aac-4d6e-afd8-6f946c6ce288.png)   |    ![image](https://user-images.githubusercontent.com/83360636/235674291-c9f06ffc-08bc-49c0-badf-d26a2a9820c7.png)  |